src/state/markets/fifo.rs

```diff
@@ -697,6 +741,13 @@ impl<
             }
         }
 
+        let (current_slot, current_unix_timestamp) = get_clock_fn();
+
+        if order_packet.is_expired(current_slot, current_unix_timestamp) {
+            phoenix_log!("Order parameters are expired, skipping matching and posting");
```

Comment:
nit: `Order parameters include a last_valid_slot or last_valid_unix_timestamp_in_seconds in the past, skipping matching and posting`